packages/runtime/src/runtime/menu.tsx

```diff
@@ -422,43 +486,100 @@ const MenuItems: React.FC<{
           borderRight: "none",
         }}
       >
-        {items.map((item, itemIndex) => (
-          <AntMenu.Item
-            data-testid={item.id ?? item.testId}
-            icon={getIcon(item)}
-            key={item.page || item.url || `customItem${itemIndex}`}
-            onClick={(): void => {
-              if (!item.openNewTab && item.page) {
-                setSelectedItem(item.page);
-              }
-            }}
-            style={{
-              color:
-                selectedItem === item.page
-                  ? (styles.selectedColor as string) ?? "white"
-                  : (styles.labelColor as string) ?? "grey",
-              display: item.visible === false ? "none" : "flex",
-              justifyContent: "center",
-              borderRadius: 0,
-              alignItems: "center",
-              fontSize:
-                selectedItem === item.page
-                  ? `${
-                      parseInt(
-                        `${styles.labelFontSize ? styles.labelFontSize : 1}` ||
-                          "1",
-                      ) + 0.2
-                    }rem`
-                  : `${styles.labelFontSize ? styles.labelFontSize : 1}rem`,
-              height: "auto",
-              ...(selectedItem === item.page
-                ? styles.onSelectStyles ?? {}
-                : {}),
-            }}
-          >
-            <CustomLink item={item}>{getLabel(item)}</CustomLink>
-          </AntMenu.Item>
-        ))}
+        {items.map((item, itemIndex) =>
+          item.children ? (
+            <AntMenu.SubMenu
+              icon={getIcon(item)}
+              key={`submenu-${itemIndex}`}
+              style={{
+                color: styles.labelColor ?? "grey",
+                fontSize: "1rem",
+                height: "auto",
+              }}
+              title={getLabel(item)}
+            >
+              {item.children.map((childItem, childIndex) => (
+                <AntMenu.Item
```

Comment:
This can be put into a function to make it more readable since this is the same as lines 546-580?